spec/workers/emails/send_user_digest_worker_spec.rb

```diff
@@ -20,29 +20,29 @@
       before { user.follow(author) }
 
       it "send digest email when there are at least 3 hot articles" do
-        articles = create_list(:article, 3, user_id: author.id, public_reactions_count: 20, score: 20)
+        create_list(:article, 3, user_id: author.id, public_reactions_count: 20, score: 20)
 
         worker.perform(user.id)
 
-        expect(DigestMailer).to have_received(:with).with(user: user, articles: articles)
+        expect(DigestMailer).to have_received(:with).with(user: user, articles: Array)
```

Comment:
This was giving me serious issues after adding the `select` statement in. I even regathered the articles here with the select statement for this assertion and despite it looking the exact same in the debugger, Rspec was still reporting them differently. I am sure it has something to do with the AR objects not being the same. Rather than fight it, after about an hour of playing around, I settled for a general assertion here with the array.

Reply:
It migth yeah, don't know. Let's keep it like that :)